modules/nf-core/ensemblvep/vep/ensemblvep-vep.diff

```diff
@@ -1,22 +1,29 @@
 Changes in module 'nf-core/ensemblvep/vep'
-'modules/nf-core/ensemblvep/vep/meta.yml' is unchanged
 Changes in 'ensemblvep/vep/main.nf':
 --- modules/nf-core/ensemblvep/vep/main.nf
 +++ modules/nf-core/ensemblvep/vep/main.nf
-@@ -1,6 +1,6 @@
+@@ -1,11 +1,11 @@
  process ENSEMBLVEP_VEP {
      tag "$meta.id"
 -    label 'process_medium'
 +    label 'process_low'
```

Comment:
why process_low?

Reply:
@ToonRosseel ?

Reply:
don't remember anymore why I changed the default. my guess: only +-200-300 variants per sample so no high memory is needed. but it's ok for me to revert it to "medium"

Reply:
in current setting, 4 cpus are used for the annotation step

Reply:
That seemed to work perfectly on the test run

Reply:
if you agree @matthdsm, I would leave it "low" for faster process planning